flaskProject/resources/reserved_start.py

```diff
@@ -91,6 +91,53 @@ def post(self):
         else:
             return {"message": "ERROR RESERVED MOTO. Motorcycle error or client not found for Reserved Moto post"}, 404
 
+    def delete(self, client_email, moto_id):
+        try:
+            moto = MotoModel.find_by_id(moto_id)
+            client = ClientModel.find_by_email(client_email)
+            # En caso de que tanto la moto y el cliente existan en sus respectivas tablas
+            if moto is not None and client is not None:
+                moto = MotoModel.find_by_id(moto_id)
+                client = ClientModel.find_by_email(client_email)
+                # Compruebo si existe el cliente segun el email recibido y si existe la moto segun el id recibido
+                if moto is not None and client is not None:
+                    reserve = ReservedRunningModel.query.filter(and_(
```

Comment:
esto es interesante, aunque con un id ya te serviria para buscarlo, no? no hace falta el and creo

Reply:
Es a propósito, quiero asegurarme de que cumple las dos condiciones y no solo una.